components/Mentor/index.jsx

```
@@ -73,32 +94,46 @@ export const Mentor = () => {
-  const handleAreaChange = (setFieldValue, area) => {
-    setFieldValue("area", area);
-    const selectedArea = areasOption.find(option => option.value === area);
+  const handleAreaChange = (setFieldValue, id) => {
+    setFieldValue("area", id);
+    const selectedArea = areas.find(option => option.id === Number(id));
+    setSelectedArea(selectedArea);
 
-    if (selectedArea && selectedArea.sub) {
-      setSubareas(selectedArea.sub);
+    if (selectedArea && selectedArea.subareas) {
+      setSubareas(selectedArea.subareas);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)